fr.openfisca.org/legislation-explorer/deploy.sh

```diff
@@ -0,0 +1,7 @@
+git pull
+npm install
+npm install swagger-ui@3.17.3
+
+NODE_ENV=production PATHNAME=/legislation/  API_URL=https://fr.openfisca.org/api/v22 npm run build
```

Comment:
How did you figure out which environment variables were needed for the build? Are we 100% sure we don't need `MATOMO_SITE_ID` here? 

As we seem to need these environment variable in 2 specific places, creating a common `.env` seems to make sense to avoid repetition. If the `.env` file is not automatically loaded as it should be, we could use a workaround:
- Use the [EnvironmentFile directive](https://coreos.com/os/docs/latest/using-environment-variables-in-systemd-units.html) in the service to source the `.env` file
- Here, `export $(cat .env | xargs) && npm run build`

but fixing https://github.com/openfisca/legislation-explorer/pull/171 maybe a more long-term solution 🤔

Reply:
> How did you figure out which environment variables were needed for the build? Are we 100% sure we don't need MATOMO_SITE_ID here?
I didn't. The legex does not function without API_URL or PATHNAME passed during the build. This was my main concern.

> Use the EnvironmentFile directive in the service to source the .env file
The `.service` is not used (as of right now) to build the project. So I don't know how to implement it so that it can source the `.env`.

> export $(cat .env | xargs) && npm run build

I like it! I will test it immediatly!

> but fixing openfisca/legislation-explorer#171 maybe a better long-term solution 

I think so too. I will implement the solution for the webpack dev configuration and ask for a review.